sdcm/utils/cloud_monitor/resources/__init__.py

```diff
@@ -7,7 +7,7 @@
 class CloudInstance:  # pylint: disable=too-few-public-methods,too-many-instance-attributes
     pricing = None  # need to be set in the child class
 
-    def __init__(self, cloud, name, instance_id, region_az, state, lifecycle, instance_type, owner, create_time, keep):  # pylint: disable=too-many-arguments
+    def __init__(self, cloud, name, instance_id, region_az, state, lifecycle, instance_type, owner, create_time, keep, project='N/A'):  # pylint: disable=too-many-arguments
```

Comment:
do we have now a default project?

Reply:
if we do, we should use it in here, instead of `N/A`

Reply:
yes, for gce (adding it in a diffrent PR), but this default is also for backends we don't have projects in, i.e. AWS